src/components/pacbio/PacbioPoolEdit.vue

```diff
@@ -146,13 +153,17 @@ export default {
         this.busy = false
       })
     },
-    async uploadFile(newFile) {
-      if (newFile === null) {
+
+    // Allows users to upload a file to autopopulate the pool's selected libraries
+    async uploadFile(evt) {
+      if (evt === null || evt.target.files === null || evt.target.files.length == 0) {
```

Comment:
Advisory:This could also be`if (!!evt?.target?.files?.length)` but that reverses the result so not sure if it works here.